src/incremental/__init__.py

```diff
@@ -391,4 +391,8 @@ def _get_version(dist, keyword, value):
 from ._version import __version__  # noqa
 
 
+def _setuptools_version():
+    return __version__.public()
+
+
```

Comment:
Interesting, the previously used `.base()` is one of three aliases for `.public()`...

https://github.com/twisted/incremental/blob/8fceca84f4d677ac497174025f11e25b6abbf110/src/incremental/__init__.py#L245-L247

If taking the suggestion above then remove this.

```suggestion
```

Reply:
yeah it doesn't work, sadly